imap_processing/tests/hit/test_housekeeping.py

```diff
@@ -1,91 +0,0 @@
-import numpy as np
```

Comment:
You can also remove the Housekeeping data class

Reply:
I did initially, but it broke my hit l1b tests for housekeeping so to avoid putting those code updates in this PR, I just left the data classes for now.